pretext/SimplePythonData/OperatorsandOperands.ptx

```diff
@@ -14,79 +14,87 @@ hour * 60 + minute
     </input>
   </program>
-  <p xml:id="simple-python-data_index-1">When a variable name appears in the place of an operand, it is replaced with
-            the value that it refers to before the operation is performed.
-            For example, what if we wanted to convert 645 minutes into hours.
-            In Python 3, division is denoted by the operator token <c>/</c> which always evaluates to a floating point
-            result.</p>
+
+<p> The first six operators have functionality that you're most probably pretty familiar with! On the other hand, we have two more operators that are used frequently in programming but not that much in math classes.
+These are integer division, and modulus. </p>
+
+<p> In the previous example <c> 5 / 3 </c> resulted <c> 1.66666666 </c>. This is because <c>/</c> always evaluates to a floating point result. However, in certain situations, we need an integer result from division.
+<term>Integer division</term>, or the <c> // </c> operator, is division such that its results are always rounded down to the nearest integer. So for the previous example, <c> 5 // 3 </c> will result <c> 1 </c> </p>
+
+<p>The <term>modulus operator</term>, <c> % </c> works on integers (and integer expressions) and yields
+            the remainder when the first operand is divided by the second (that's why you can also remember it as remainder operator). For example, <c> 5 % 3 </c> will result <c> 2 </c> as   <c> 5 = 3 * 1 + 2 </c>. 
+            The modulus operator can only give values between 0 and second integer minus 1. </p>
+
   <program xml:id="ch2_sec7_p2" interactive="activecode" language="python">
     <input>
-minutes = 645
-hours = minutes / 60
-print(hours)
+print(5//3) # Integer division operator, expected result is 1
+print(60//7) # Integer division operator, expected result is 8
+print(50//2) # Integer division operator works same as division operator when the result is integer
+
+print(5%3) # Modulus operator, expected result is 2
+print(60%7) # Modulus operator, expected result is 4
+
+# Modulus operator can only give values between 0 and second integer-1.
+print(3%3) # 0
+print(4%3) # 1 
+print(5%3) # 2 (3-1)
+print(6%3) # Again 0!
+
     </input>
   </program>
-  <p>What if, on the other hand, we had wanted to know how many <em>whole</em> hours there
-            are and how many minutes remain.  To help answer this question, Python gives us a second flavor of
-            the division operator.  This version, called <term>integer division</term>, uses the token
-    <c>//</c>.  It always <em>truncates</em> its result down to the next smallest integer (to
-            the left on the number line).</p>
+
+   <p xml:id="simple-python-data_index-1">When a variable name appears in the place of an operand, it is replaced with
+            the value that it refers to before the operation is performed.
+            For example, what if we wanted to convert 645 minutes into hours?
```

Comment:
This sentence is really confusingly worded.

When a variable name appears in place of an operand, python substitutes that variable name with its assigned value.

Reply:
changed!